user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+
+
+* Using Easy-RSA configuration:
+  /kubestellar/pki/vars
+
+* Using SSL: openssl OpenSSL 3.0.7 1 Nov 2022 (Library: OpenSSL 3.0.7 1 Nov 2022)
+
+-----
+
+Notice
+------
+Keypair and certificate request completed. Your files are:
+* req: /kubestellar/pki/reqs/kcp-server.req
+* key: /kubestellar/pki/private/kcp-server.key
+
+Using configuration from /kubestellar/pki/0525cd36/temp.5.1
+Check that the request matches the signature
+Signature ok
+The Subject's Distinguished Name is as follows
+commonName            :ASN.1 12:'kcp-server'
+Certificate is to be certified until Dec 16 21:39:38 2050 GMT (10000 days)
+
+Write out database with 1 new entries
+Data Base Updated
+
+Notice
+------
+Certificate created at:
+* /kubestellar/pki/issued/kcp-server.crt
+
+Notice
+------
+Inline file created:
+* /kubestellar/pki/inline/kcp-server.inline
+
+Server=kubestellar.svc.cluster.local
+/kubestellar/pki/ca.crt
+/kubestellar/pki/issued/kcp-server.crt
+/kubestellar/pki/private/kcp-server.key
+< Starting kcp >-------------------------------------------
+Running kcp... logfile=./kubestellar-logs/kcp.log
+Waiting for kcp to be ready... it may take a while
+kcp version: v0.11.0
+Current workspace is "root".
+Switching the admin.kubeconfig domain to kubestellar.svc.cluster.local...
+< Starting KubeStellar >-----------------------------------
+Finished augmenting root:compute for KubeStellar
+Workspace "espw" (type root:organization) created. Waiting for it to be ready...
+Workspace "espw" (type root:organization) is ready to use.
+Current workspace is "root:espw" (type root:organization).
+Finished populating the espw with kubestellar apiexports
+****************************************
+Launching KubeStellar ...
+****************************************
+ mailbox-controller is running (log file: /kubestellar/kubestellar-logs/mailbox-controller-log.txt)
+ where-resolver is running (log file: /kubestellar/kubestellar-logs/kubestellar-where-resolver-log.txt)
+ placement translator is running (log file: /kubestellar/kubestellar-logs/placement-translator-log.txt)
+****************************************
+Finished launching KubeStellar ...
+****************************************
+Current workspace is "root".
+< Create secrets >-----------------------------------------
+Ensure secret in the curent namespace...
+secret/kubestellar created
+Ensure secret in namespace "default"...
+secret "kubestellar" deleted
+secret/kubestellar created
+Ready!
+```
+
+Note that alternatively, one can wait for the `kubestellar` secret to be created.
+
+```shell
+kubectl wait -n kubestellar \
+  --for=condition=ready pod \
+  --selector=app=kubestellar-server \
+  --timeout=120s
+
+until kubectl logs -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) 2>&1 | grep -Fxq "Ready!"
+do
+  sleep 1
+done
+```
+
+After the deployment has completed, **KubeStellar** `admin.kubeconfig` can be in three ways:

```

Comment:
"can be" _what_ "in three ways"?

Reply:
Sorry I removed one of the ways... fixed.